src/main/java/org/stellar/sdk/responses/GsonSingleton.java

```diff
@@ -1,14 +1,16 @@
 package org.stellar.sdk.responses;
 
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
+import com.google.gson.*;
 import com.google.gson.reflect.TypeToken;
 
 import org.stellar.sdk.Asset;
 import org.stellar.sdk.KeyPair;
 import org.stellar.sdk.responses.effects.EffectResponse;
 import org.stellar.sdk.responses.operations.OperationResponse;
 
+import java.lang.reflect.Type;
+import java.util.Base64;
```

Comment:
AFAIR, this package does not work in Android. Please use `com.google.common.io.BaseEncoding`.

Reply:
ok I look into it